### PR TITLE
refactor : update card style for mobile view

### PR DESCRIPTION
**Pull request by viru2001**
## change log

made the card on cart page responsive 

responsive cart card

[Deploy Preview](https://deploy-preview-17--gamers-zone.netlify.app/pages/cart.html)

### CODE REVIEW

File: styles/cart.css [hunk 1]
```diff
@@ -76,11 +80,19 @@ footer {
 }
 
 @media (max-width:624px) {
-  .cart-btn-wrapper{
-    gap : 2rem;
-    display: flex;
+  .card-wrapper{
+    flex-direction: column;
+  }
+  .cart-action-container{
```
Comment (mohitdhatrak) [inline styles/cart.css:86]: i think flex direction column has no impact here maybe, try removing. Might be wrong too

**Review by mohitdhatrak — COMMENTED** [review]
Just few suggestions, merge!

**Review by mohitdhatrak — COMMENTED** [review]
Also, one more IMP thing- the save 40% etc text in green is having contrast issues i checked.

File: styles/cart.css (hunk 1 above)
Comment (viru2001) [reply styles/cart.css:86]: it is useful. see in small screen product image and product details are in column

**Comment by viru2001** [comment]
> Also, one more IMP thing- the save 40% etc text in green is having contrast issues i checked.

ohhkk. Thank you for the suggestion. I will change it everywhere now

**Review by bhavyaj12 — APPROVED** [review]
Looks great!

**Pull request merged**